### PR TITLE
The Ember Times No. 175 - March 12th 2021

### PR DESCRIPTION
**Pull request by amyrlam**
👉 New to The Ember Times? If you want to claim a topic to write about, please comment here on the PR or chat with us in [#support-ember-times](https://discordapp.com/channels/480462759797063690/485450546887786506) on the [Ember Discord](https://discord.gg/emberjs)! 

👉 New thing If you want rich content "inline" in Substack, or a particular thing in our ICYMI tweet, please comment on this PR with a: 
`- [ ] add tweet / youtube link`

***

### Addon/App releases

- [x] Just released ember-responsive-image v2 https://twitter.com/simonihmig/status/1369316435621724163 from https://discord.com/channels/480462759797063690/480499624663056390/818875541927100466 (🔒 @ijlee2)
- [ ] We just published ember-template-lint@3.0.0 today! Read more about it in the release notes: https://github.com/ember-template-lint/ember-template-lint/releases/tag/v3.0.0. Please test and file issues (or pull requests :wink: ) for anything you run into! from https://discord.com/channels/480462759797063690/480499624663056390/816368004207476806 (NOTE: we've written about it before, but if something new(?))
- [ ] `ember-oauther` release
    - https://twitter.com/sinankesk_in/status/1369032321890988033
- [ ] v0.2.0 of ember-resize-modifier from https://discord.com/channels/480462759797063690/480499624663056390/816792500135133265

### Articles

- [x] [How to handle uploading and parsing files in your frontend app tests](https://twitter.com/herzzanu/status/1359968424169009152). Just wrote this article on how we mock uploading, reading and parsing files in our tests using Mirage at @getqonto #emberjs #javascript https://twitter.com/herzzanu/status/1359968424169009152 (:lock: @amyrlam)
- [ ] We just published a new blog post on using #serverless functions with #emberjs and #glimmer. If you have used API routes in #nextjs this is a similar concept that can be applied to other #javascript frameworks! https://twitter.com/shipshapecode/status/1363915759169208320 from #news-and-announcements https://discord.com/channels/480462759797063690/480499624663056390/813474766261321759
- [ ] Achieving A11y Automation Testing, with Ava Wroten
    - https://twitter.com/TestJSSummit/status/1369241608944762883

### Readers' Questions / Miscellaneous

- [x] The annual community survey is up- take the survey today and let us know what you think about Ember! https://twitter.com/emberjs/status/1366794720761028611 and https://discord.com/channels/480462759797063690/480499624663056390/816353623469981706
- [x] Episode 2 of The Ember Report, with David Tang 
    - https://twitter.com/EmberReport/status/1369010817518166020 (🔒  @jaredgalanis)
- [x] [RFC #724: Official TypeScript support](https://github.com/emberjs/rfcs/pull/724) (🔒 @chrisrng )
- [x] Contributors workshop @locks 

- [ ] Idea: A writeup about International Women's Day and how Ember is committed to inclusiveness

## 🤔  Questions?

See #support-ember-times on Discord or CONTRIBUTING.

## ✅  Publishing TODOs:

- [x] Check Netlify / localhost preview, to ensure display e.g. buttons on emberjs.com/blog is good
- [x] Crosspost on dev.to: delete any alex ignore's, manually redo Readers Question picture, add Canonical URL ... set up and try for Mon post instead
- [x] Set up tweet for Monday morning on tweetdeck
- [x] Post $natural for #news-and-announcements in Discord

### CODE REVIEW

File: content/the-ember-times-issue-175.md [hunk 1]
```diff
@@ -11,22 +12,26 @@ tags:
...
+[Chris Krycho (@chriskrycho)](https://github.com/chriskrycho) put up an [RFC for Official TypeScript support](https://github.com/emberjs/rfcs/pull/724) in Ember. The goal of the RFC is to determine if we should support [TypeScript](https://www.typescriptlang.org/) as a first-class citizen in Ember, and if yes, identify any gaps present today.
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+> Ember officially supporting TypeScript means: All libraries which are installed as part of the default blueprint must ship accurate and up-to-date type definitions for the current edition. These types will uphold a Semantic Versioning commitment which includes a definition of SemVer for TypeScript types as well as a specification of supported compiler versions and settings, so that TypeScript will receive the same stability commitments as the rest of Ember.
+
+- Any addon that is part of the default blueprint must ship types.
+- The RFC explicitly does not require support for “typed templates”
+- Instead of opting into TypeScript using [ember-cli-typescript](https://github.com/typed-ember/ember-cli-typescript) it should be done by default
```
Comment (chriskrycho) [inline content/the-ember-times-issue-175.md]: Note: this is actually an open question! We should *support* it by default, but it's quite possible that we'll still be using (an updated version of) ember-cli-typescript for it just like we support Babel "out of the box" but do so via ember-cli-babel. (My personal opinion is that that's *likely*, in fact, but we'll need to hash it out in a future RFC!)

File: content/the-ember-times-issue-175.md [hunk 2]
```diff
@@ -11,22 +12,26 @@ tags:
...
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Chris Krycho (@chriskrycho)](https://github.com/chriskrycho) put up an [RFC for Official TypeScript support](https://github.com/emberjs/rfcs/pull/724) in Ember. The goal of the RFC is to determine if we should support [TypeScript](https://www.typescriptlang.org/) as a first-class citizen in Ember, and if yes, identify any gaps present today.
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+> Ember officially supporting TypeScript means: All libraries which are installed as part of the default blueprint must ship accurate and up-to-date type definitions for the current edition. These types will uphold a Semantic Versioning commitment which includes a definition of SemVer for TypeScript types as well as a specification of supported compiler versions and settings, so that TypeScript will receive the same stability commitments as the rest of Ember.
+
+- Any addon that is part of the default blueprint must ship types.
+- The RFC explicitly does not require support for “typed templates”
```
Comment (chriskrycho) [inline content/the-ember-times-issue-175.md]: I'd love it if we clarified here that it's also a thing the community is actively working on. It's good to be clear that we aren't *blocked* on it, but we also think it's absolutely key to the long-term experience of TS in Ember!

File: content/the-ember-times-issue-175.md (hunk 1 above)
Comment (amyrlam) [reply content/the-ember-times-issue-175.md]: @chriskrycho Hi Chris, I'm working on tying up loose ends for publish tonite. If you have time, can you please submit a GitHub suggestion / PR for the changes you want? No worries if not, I will take my best stab at it later on.

Comment (chriskrycho) [reply content/the-ember-times-issue-175.md]: I'll submit a PR in a couple minutes!

Comment (chriskrycho) [reply content/the-ember-times-issue-175.md]: See #933!

**Pull request merged**